src/inmanta/ast/statements/define.py

```diff
@@ -141,8 +140,6 @@ def __init__(
         self.attributes = attributes
         if comment is not None:
             self.comment = str(comment)
-        else:
-            self.comment = ""
```

Comment:
I think we should still set it. If I understand correctly you're currently counting on the default, but that's a class variable rather than an instance variable (for `pydantic` and `dataclasses` we do it like that because they do the magic for us). I think it should work like this (but I'm not even sure) but it is definitely good practice to set it on the instance.

Same for the classes below.

Reply:
Ah no, we don't inherit from `WithComment` so there isn't even the class var.